packages/react/src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -603,44 +623,29 @@ const SelectPanelMessage: React.FC<SelectPanelMessageProps> = ({
         {variant !== 'empty' ? (
           <Octicon icon={AlertIcon} sx={{color: variant === 'error' ? 'danger.fg' : 'attention.fg', marginBottom: 2}} />
         ) : null}
-        <Text sx={{fontSize: 1, fontWeight: 'semibold'}}>{title}</Text>
+        <Text sx={{fontSize: 1, fontWeight: 'semibold'}}>{title} </Text>
```

Comment:
curious little space 😀

Reply:
Haha, let me know if there's a good way to document this. This space is needed so that that title and message aren't merged together when announced (thankfully the tests should cover this too if it's ever removed in the future). By default, the text content of the container will end up being `titlemessage` instead of `title message`.

Reply:
Ah i see, that's clever! 

I've also used a visually hidden comma in the past 😇